packages/widget/src/pages/TransactionHistoryPage/TransactionHistoryPageHistoryItemDetails.tsx

```diff
@@ -50,12 +52,70 @@ export const TransactionHistoryPageHistoryItemDetails = ({
     return;
   }, [status, theme]);
 
-  const handleClickingLinkIfNoExplorerLink = (txHash: string, explorerLink?: string) => {
+  const handleClickingLinkIfNoExplorerLink = (txHash?: string, explorerLink?: string) => {
     if (!explorerLink) {
       copyToClipboard(txHash);
     }
   };
 
+  const getTxHashFromLink = (link?: string) => {
+    const splitLinkBySlash = link?.split("/");
+    if (!splitLinkBySlash) return;
+    return splitLinkBySlash[splitLinkBySlash.length - 1];
+  };
+
+  const renderTransactionIds = useMemo(() => {
+    const initialTxHash = transactionDetails?.[0]?.txHash;
+
+    if (explorerLinks?.length === 0) {
```

Comment:
```suggestion
    if (explorerLinks?.length === 1 {
```

Reply:
this case is for handling 0 explorerLinks

Reply:
widget/constants/cosmosChains/explorer.json

Reply:
use `createExplorerLink` in explorerLink.ts